packages/nouns-webapp/src/locales/en-US.js

```diff
@@ -0,0 +1,317 @@
+/*eslint-disable*/ module.exports = {
```

Comment:
The build catalogs typically should not be committed. Add to the `.gitignore` and compile in the `prepare` hook instead.